### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by burf2000** [ginge/Hector#2]
So are you carrying on with this mate?

**Comment by ginge** [comment]
Yeah slowly I guess. Not much happened in a few months, but I did start
redesigning some parts recently

Cheers for the interest
